packages/button/test/button.test.js

```diff
@@ -61,21 +61,9 @@ describe('vaadin-button', () => {
       expect(children[2].outerHTML).to.be.equal('<i>Button</i>');
     });
 
-    describe('disabled', () => {
-      beforeEach(() => {
-        element.disabled = true;
-      });
-
-      it('should disable the native button', () => {
-        expect(button.hasAttribute('disabled')).to.be.eql(true);
-      });
-
-      it('should not fire the click event', () => {
-        const spy = sinon.spy();
-        element.addEventListener('click', spy);
-        element.click();
-        expect(spy.called).to.be.false;
-      });
```

Comment:
Decided to remove this test as well as [here](https://github.com/vaadin/web-components/pull/2354#discussion_r692036777) since there is no logic relevant to `click` listeners anymore.